Animations/Fade.js

```diff
@@ -1,6 +1,8 @@
-var Animation = require('./Animations.js');
+var makeAnimation = require('./Animation');
 //Defult Fade animation component
 var Fade = {
+  //Class name given to the animation component once mounted
+  className: 'fade',
   //CSS for wrapped component on entry
   enter: {
     'opacity': '0.01',
```

Comment:
Is there anyway we can store some of the css props that are common between all of our animations in an object in a separate file and then use `object-assign` to build out our individual animations by extending this base object.  Right now our code seems kind of wet since every animation has `enter: {'opacity': '0.01'...`.